lms/templates/admin/instance.html.jinja2

```diff
@@ -136,6 +141,7 @@
         {{ settings_text_field("JSTOR site code", "jstor", "site_code") }}
     </fieldset>
 
+    {{ settings_textarea("Notes", "hypothesis", "notes") }}
```

Comment:
I reckon the notes should go at the very top or bottom (as they are easier to spot/go to).

Here at the bottom I reckon they fit better with some other "meta" fields like the dates.